README.md

```diff
@@ -43,54 +46,27 @@ Contents:
 
 <!-- TOC -->
 
+- [How to build Valora](#build)
```

Comment:
Recommend moving this section down to after The Valora Stack

Reply:
sure!